### PR TITLE
Update css background size documentation with images

### LINKED ISSUE #3637: [Edit] CSS [Edit] CSS Background: background-size
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to a term within CSS Background. The term entry lives in docs/content/css/concepts/background/terms/background-size/background-size.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by laisvigas**
### Description

Closes #3637

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by laisvigas** [comment]
Hi, @SSwiniarski I wanted to inform you that I've addressed the issues encountered in the last pull request, and I've created a new one. I kindly request that you or someone from the team to review the changes and, if everything is satisfactory, proceed with the merge. Thank you!

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@laisvigas You need to address the extra file in this PR.

File: content/lua/concepts/mathematical-library/terms/asin/asin.md
[the remark this answers is not in the document]
Comment (laisvigas) [reply content/lua/concepts/mathematical-library/terms/asin/asin.md:1]: Done.

**Review by SSwiniarski — APPROVED** [review]
@laisvigas looks good for another review.

**Review by caupolicandiaz — APPROVED** [review]
@laisvigas @SSwiniarski LGTM

**Pull request merged**